.github/workflows/md5_check_initial-data.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Please add a comment that this is only for protecting against unwanted/unintended edits. But if one is sure to change this file it is allowed!

Reply:
done